site/en/docs/devtools/settings/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Settings overview"
+authors:
+  - sofiayem
+date: 2023-02-14
+#updated: YYYY-MM-DD
+description: "Settings overview."
+---
+
+DevTools settings let you control the behavior of both individual panels and DevTools in general.
+
+The {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} **Settings** panel has several tabs described in the following sections:
+
+- [**Preferences**](/docs/devtools/settings/preferences/)
+- [Other tabs](/docs/devtools/settings/other/): **Workspace**, **Experiments**, **Ignore List**, **Devices**, **Throttling**, **Locations**, and **Shortcuts**
+
+## Open Settings {: #open }
+
+To open {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} **Settings**:
+
+1. [Open DevTools](/docs/devtools/open/) on any page.
+1. Click the {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/9gzXiTYY0nZzBxGI6KrV.svg", alt="Settings.", width="24", height="24" %} **Settings** button in the action bar at the top.
+
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/KSgG5FtMU6pvdR8qUpJq.png", alt="The Settings button in the action bar at the top of DevTools.", width="800", height="464" %}
+
+   {% Aside 'gotchas' %}
+   Be careful not to confuse general DevTools settings with panel settings. DevTools settings are on the topmost action bar.
+
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/OQk9CIpYLf4lGToIyL5G.png", alt="General and panel settings.", width="800", height="315" %}
```

Comment:
I saw a couple of images in this PRs using label text within image. A reminder on that, avoid using label text in image whenever possible, for accessibility purpose. 

Not sure if this mentioned in the styles guide, if it's not this should be added in.

Reply:
I find that, in this case, it makes sense for people who can see. I'll add a better alt text for a11ty.